.github/workflows/diff_protobin.yml

```diff
@@ -5,6 +5,7 @@ on:
     paths:
       - '**/*.bin'
       - '**/*.guildpoint'
+    types: [opened, synchronize, ready_for_review]
```

Comment:
Lets add a description of what this is doing, ie how is it changing the default values. Maybe even a link to the docs for it in case the default value changes in the future.